### PR TITLE
Codestyle

### PR DESCRIPTION
**Pull request by michael-k**
This is my first contribution to this project, so I'm not sure what's appreciated and what's not. Please let me know. I'm happy to change the PR in any way necessary.

As I worked myself through the codebase I made some changes to the codestyle and did some minor refactoring (switched if-else; returned nil as payload with error). I split it in small commits to make it easier to follow.

### CODE REVIEW

**Comment by technoweenie** [comment]
Looks good to me.  There's a lot of old Go code of mine in here written before I understood the popular Go conventions.

File: commands/commands.go [hunk 1]
```diff
@@ -12,6 +10,9 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
+
+	"github.com/github/git-lfs/lfs"
+	"github.com/spf13/cobra"
```
Comment (rubyist) [inline commands/commands.go]: If we're going to go with this style of import block (which I am in favor of, and is the only annoying difference between `goimports` and `go fmt`), then we should do it this way everywhere.

**Comment by rubyist** [comment]
:+1: for me, other than hoping to see those import blocks consistent across the code base.

**Comment by michael-k** [comment]
Thanks for the feedback. I'll change the imports.

**Comment by technoweenie** [comment]
I don't think the imports are necessary to change in this PR.  Instead, I think we should setup a separate PR that somehow bundles `goimports`, and uses it in place of `script/fmt`.  I'd probably just add the `go get` in `script/bootstrap`.

**Comment by rubyist** [comment]
I think this PR should leave them consistent, whichever way we go. I'm not entirely convinced bundling goimports is necessary, go fmt will honor the spaced blocks if it encounters them. I could be persuaded otherwise, though. I honestly don't use script/fmt and just run that stuff from my editor.

**Comment by technoweenie** [comment]
Actually, you do use script/fmt, it's called during script/test :)

I'm totally cool with consistent imports in this PR.

On Tue, May 12, 2015 at 10:25 AM, Scott Barron <notifications@github.com>
wrote:

> I think this PR should leave them consistent, whichever way we go. I'm not
> entirely convinced bundling goimports is necessary, go fmt will honor the
> spaced blocks if it encounters them. I could be persuaded otherwise,
> though. I honestly don't use script/fmt and just run that stuff from my
> editor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/github/git-lfs/pull/300#issuecomment-101339163>.
>



-- 
Rick Olson

**Comment by michael-k** [comment]
I removed the import reformatting from this PR. (That's what I meant with “I'll change the imports.”)

**Comment by technoweenie** [comment]
Thanks!  I think we'll just do that in an isolated PR.

**Pull request merged**